apps/web/pages/signup.tsx

```diff
@@ -83,7 +82,7 @@ export default function Signup({ prepopulateFormValues, token, orgSlug }: Signup
   };
 
   return (
-    <LicenseRequired>
```

Comment:
LicenseRequired only works for logged in users. All users landing will be logged off.

Reply:
we dont need this for this component anyway. great catch!